py/frequenz/client/weather/_types.py

```diff
@@ -281,3 +281,85 @@ def to_ndarray_vlf(
             raise RuntimeError("Error processing forecast data") from e
 
         return array
+
+
+@dataclass(frozen=True)
+class HistoricalForecasts:
+    """Historical weather forecast data."""
+
+    _forecasts_pb: weather_pb2.GetHistoricalWeatherForecastResponse
```

Comment:
Thank you for raising the awareness of the ongoing issue. Since the issue is not resolved, I suggest to keep it this way as it is. It is consistent with Forecasts class and classes for other protobuf objects (e.g. LocationForecast) that already exist here. Once the issue is resolved and conventions change, we can change both Forecasts and HistoricalForecasts classes. I created an issue for this: https://github.com/frequenz-floss/frequenz-api-weather/issues/95